bot/seasons/evergreen/issues.py

```diff
@@ -3,55 +3,60 @@
 import discord
 from discord.ext import commands
 
-from bot.constants import Colours
-from bot.decorators import override_in_channel
-
 log = logging.getLogger(__name__)
 
+icons = {"issue": "https://i.imgur.com/HFV0nv9.png",
+         "issue-closed": "https://i.imgur.com/uZSX7as.png",
+         "pull-request": "https://i.imgur.com/zHhnALX.png",
+         "pull-request-closed": "https://i.imgur.com/JYmTn5P.png",
+         "merge": "https://i.imgur.com/xzQZxXe.png"}
+
+respValue = {404: "Issue/pull request not located! Please enter a valid number!",
+             403: "rate limit has been hit! Please try again later!"}
+
 
 class Issues(commands.Cog):
     """Cog that allows users to retrieve issues from GitHub."""
 
     def __init__(self, bot: commands.Bot):
         self.bot = bot
 
-    @commands.command(aliases=("issues",))
-    @override_in_channel()
-    async def issue(
-        self, ctx: commands.Context, number: int, repository: str = "seasonalbot", user: str = "python-discord"
-    ) -> None:
-        """Command to retrieve issues from a GitHub repository."""
-        api_url = f"https://api.github.com/repos/{user}/{repository}/issues/{number}"
-        failed_status = {
-            404: f"Issue #{number} doesn't exist in the repository {user}/{repository}.",
-            403: f"Rate limit exceeded. Please wait a while before trying again!"
-        }
-
-        async with self.bot.http_session.get(api_url) as r:
-            json_data = await r.json()
-            response_code = r.status
-
-        if response_code in failed_status:
-            return await ctx.send(failed_status[response_code])
-
-        repo_url = f"https://github.com/{user}/{repository}"
-        issue_embed = discord.Embed(colour=Colours.bright_green)
-        issue_embed.add_field(name="Repository", value=f"[{user}/{repository}]({repo_url})", inline=False)
-        issue_embed.add_field(name="Issue Number", value=f"#{number}", inline=False)
-        issue_embed.add_field(name="Status", value=json_data["state"].title())
-        issue_embed.add_field(name="Link", value=json_data["html_url"], inline=False)
+    @commands.command(aliases=("pr",))
+    async def issues(self, ctx: commands.Context, number: int, repository: str = "seasonalbot",
```

Comment:
`issue` was probably a better command name, `issues` implies that multiple issues can be fetched, which isn't supported.